pkgs/os-specific/linux/sysdig/default.nix

```diff
@@ -3,14 +3,28 @@ let
   inherit (stdenv.lib) optional optionalString;
   s = rec {
     baseName="sysdig";
-    version = "0.10.0";
+    version = "0.11.0";
     name="${baseName}-${version}";
     url="https://github.com/draios/sysdig/archive/${version}.tar.gz";
```

Comment:
Looks like this ends up being passed to `fetchurl`; please consider `fetchFromGitHub` instead, it'll be more stable wrt. to changes in the way github generates the archive.